views/partials/schedule.hbs

```diff
@@ -205,11 +215,13 @@
 
           <li class="event-schedule-item">
             <span class="event-schedule-dot"></span>
-            <span class="event-schedule-item-time">6:00 PM - 11:00 PM</span>
+            <span class="event-schedule-item-time">5:45 PM - ??:?? PM</span>
             <h4 class="event-schedule-item-title">
-              Closing Party on historic Stone Street
+              EmpireJS Scavenger Hunt
             </h4>
-            <p>Food, drinks, good music, and goodbyes to new friends while chatting about all things JavaScript</p>
+            <p>As the lights come up and we take our EmpireJS family photo, we’ll immediately segway into our final adventure of EmpireJS 2016—a scavenger hunt!</p>
+
+            <p>Experience our conference neighborhood and the history of NYC with your new friends! Our partners at <a href="http://museumhack.com/">Museum Hack</a> will being leading us from the stage of NMAI and around the area on an interactive scavenger hunt to interact with the FiDi and old New York. Discover surprising facts, some cool history, and maybe find a cool prize at the end! This party wraps up at a secret bar to be determined as part of our hunt!</p>
```

Comment:
Typo: "will be leading"

Also, maybe "an interactive scavenger hunt to explore FiDi and old New York", just for variety's sake